### ISSUE TITLE
REQUEST: New membership for yg

### ISSUE THREAD

**Issue by yg** [kubernetes/org#1692]
### GitHub Username

@yg

### Organization you are requesting membership in

@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors

- @savitharaghunathan

### List of contributions to the Kubernetes project

Tamil Translation https://github.com/kubernetes/website/pull/20550

**Comment by yohix** [comment]
cc: @savitharaghunathan

**Comment by savitharaghunathan** [comment]
@yohix , I cannot sponsor as I don't qualify yet. I think you would need two reviewers/approvers sponsoring your request - https://github.com/kubernetes/community/blob/master/community-membership.md#requirements.

**Comment by yohix** [comment]
@savitharaghunathan, oops how can I notify them? or else they will get back to this issue?

**Comment by cblecker** [comment]
/lifecycle stale

Is this still active? If so, please `/remove-lifecycle stale`
Two sponsors are required for Kubernetes membership. They must have close interaction, and be familiar with your work. More details are available here: https://github.com/kubernetes/community/blob/master/community-membership.md#requirements

**Comment by yohix** [comment]
@cblecker I think there are no sponsors for **Tamil**

**Comment by yohix** [comment]
It seems it takes too long to become a member! We can close this issue.

**Comment by mrbobbytables** [comment]
Thanks for letting us know @yg please reopen or create a new issue once you've found a second sponsor.

/close

**Comment by yg** [comment]
@mrbobbytables There is no active people in Tamil!

WDYT?

**Comment by savitharaghunathan** [comment]
@yg : I can review Tamil documentation and sponsor if you are still interested. However, I cannot actively engage in localization efforts due to my other commitments. Here's a doc on how to get started with localization - https://kubernetes.io/docs/contribute/localization/ and if you have any queries regarding the same, post it in https://kubernetes.slack.com/messages/sig-docs. A lot of localization efforts are going on and it is a good place for asking any docs related queries.

**Comment by yg** [comment]
Sounds good @savitharaghunathan! Let me reopen this issue :blue_heart: 

/cc @mrbobbytables

**Comment by savitharaghunathan** [comment]
@yg I will be happy to sponsor once there is a PR for Tamil localization is in place.

**Comment by yg** [comment]
Sure @savitharaghunathan I will work on some basic stuff for now and make a PR and update you :smiley:

**Comment by yg** [comment]
@savitharaghunathan I just completed translating the README to Tamil, here is my PR https://github.com/kubernetes/website/pull/20550

**Comment by savitharaghunathan** [comment]
+1  for @yg . Good luck with Tamil localization :)

**Comment by yg** [comment]
Thank you @savitharaghunathan! Let's rock it!

**Comment by sftim** [comment]
There's a fairly important requirement to have opened 5 PRs (or worked on a mixture of PRs and issues). You _don't_ have to have got the PRs merged, but you do need to show a level of involvement.

Please feel free to update the issue description with details of those other PRs.

**Comment by yg** [comment]
@sftim I updated the PR link to https://github.com/kubernetes/website/pull/20550

**Comment by nikhita** [comment]
As there has been no traction with this in a month I'm going to flag it as rotten.

/lifecycle rotten

Please remember that you must speak to your sponsors first and get their approval before applying for org membership.

If this is still active, please go ahead and /remove-lifecycle rotten

**Comment by yg** [comment]
Thanks @nikhita, we can close this!
